addons/event/models/event_ticket.py

```diff
@@ -80,6 +80,8 @@ def default_get(self, fields):
     seats_used = fields.Integer(string='Used Seats', compute='_compute_seats', store=False)
     is_sold_out = fields.Boolean(
         'Sold Out', compute='_compute_is_sold_out', help='Whether seats are not available for this ticket.')
+    # reports
+    color = fields.Char('Color', default="#875A7B")
```

Comment:
Do we need a custom color, in addition to report colors and company email colors :D ?